aries_cloudagent/config/logging.py

```diff
@@ -1,30 +1,57 @@
 """Utilities related to logging."""
-import asyncio
+
+from contextvars import ContextVar
 from datetime import datetime, timedelta
 from io import TextIOWrapper
 import logging
-from logging.config import fileConfig
+from logging.config import fileConfig, dictConfig
 from logging.handlers import BaseRotatingHandler
 import os
 from random import randint
 import re
 import sys
+import yaml
 import time as mod_time
-from typing import Optional, TextIO
+from typing import TextIO
 
 import pkg_resources
 from portalocker import LOCK_EX, lock, unlock
 from pythonjsonlogger import jsonlogger
 
 from ..config.settings import Settings
-from ..core.profile import Profile
 from ..version import __version__
-from ..wallet.base import BaseWallet, DIDInfo
 from .banner import Banner
-from .base import BaseSettings
-
 
 DEFAULT_LOGGING_CONFIG_PATH = "aries_cloudagent.config:default_logging_config.ini"
+DEFAULT_PER_TENANT_LOGGING_CONFIG_PATH = (
+    "./aries_cloudagent/config/default_per_tenant_logging_config.yml"
+)
+LOG_FORMAT_FILE_ALIAS_PATTERN = (
+    "%(asctime)s %(wallet_id)s %(levelname)s %(pathname)s:%(lineno)d %(message)s"
+)
```

Comment:
This looks like the same pattern/variable that is held in `default_per_tenant_logging_config.ini` - any reason to have the setting it repeated here?